extension/icu/icu-dateadd.cpp

```diff
@@ -31,6 +31,26 @@ struct ICUCalendarAge : public ICUDateFunc {
 	}
 };
 
+static inline void CalendarAddHour(icu::Calendar *calendar, int64_t interval_hour, UErrorCode &status) {
```

Comment:
Good catch!  I may move this to `ICUDateFunc` and make it take the part code, but this is good for now.